src/toniarts/openkeeper/game/controller/GameWorldController.java

```diff
@@ -329,10 +332,10 @@ public void build(Vector2f start, Vector2f end, short playerId, short roomId) {
 
         // If this is a bridge, we only got the starting point(s) as valid so we need to determine valid bridge pieces by our ourselves
         Room room = kwdFile.getRoomById(roomId);
-        if ((room.getFlags().contains(Room.RoomFlag.PLACEABLE_ON_WATER))
-                || room.getFlags().contains(Room.RoomFlag.PLACEABLE_ON_LAVA)) {
-            instancePlots = new ArrayList(mapController.getTerrainBatches(instancePlots, x1, x2, y1, y2));
-        }
+//        if ((room.getFlags().contains(Room.RoomFlag.PLACEABLE_ON_WATER))

```

Comment:
Obsolete now? Shouldn't have code in comments, unless that is commented, otherwise very confusing later

Reply:
Must be deleted